ci/generate.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "--- Check Examples files"
+./examples/run_djinni.sh
+./perftest/run_djinni.sh
+./test-suite/run_djinni.sh
+
+if test -z "$(git status --porcelain)"; then
+  echo "--- Success! Examples files are clean."
```

Comment:
Can we change "Example files" to "generated files"?

Reply:
👌